CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Added the ability to connect to a local deployment when launching on a device by checking "Connect to a local deployment" and specifying the local IP of your computer in the Launch dropdown.
 - The Spatial GDK now default enables RPC Ring Buffers, and the legacy RPC mode will be removed in a subsequent release.
 - The `bPackRPCs` property has been removed, and the flag `--OverrideRPCPacking` has been removed.
+- Handover properties will be automatically replicated when required for load balancing. `bEnableHandover` is off by default.
```

Comment:
Once offloading becomes a LBS, I feel we can remove `bEnableHandover` entirely. Thoughts?

Reply:
Solid maybe. :) It seems like a handy setting for debugging, although maybe we hide it from the editor.